misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/utils/EvaluationUtils.java

```diff
@@ -77,7 +78,7 @@
     public static final String B_TYPE_ARRAY_CLASS = RUNTIME_HELPER_PREFIX + "api.types.Type[]";
     private static final String B_LINK_CLASS = RUNTIME_HELPER_PREFIX + "api.values.BLink";
     private static final String B_ERROR_VALUE_CLASS = RUNTIME_HELPER_PREFIX + "internal.values.ErrorValue";
-    public static final String FROM_STRING_CLASS = "org.ballerinalang.langlib.xml.FromString";
+
```

Comment:
Shall we remove this extra line?

Reply:
Fixed in ae66fd25bdd34c10affa694460255a0f3bf4983f